Feature/9

Atom098890 opened the pull request.

Commit:
add exchenge function for bank

Commit:
add crypto and check function

Commit:
add fuction sort for arrays

korbutds commented on 7-converter/homework.js:
```diff
@@ -0,0 +1,20 @@
+        return sum * EUR;
+    }
+
+    return null;
```
Попробуйте использовать `switch..case`. 
Подсказка: Попробуйте использовать отношение строк `currnecyFrom/currencyTo`

korbutds commented on 9-sort-loops/homework.js:
```diff
@@ -0,0 +1,18 @@
+                let numderI = arr[i];
+                let numberJ = arr[j];
+                arr[i] = numberJ;
+                arr[j] = numderI;
```
Попробуйте использовать трюк обмена переменных. 
https://learn.javascript.ru/destructuring-assignment

korbutds replied:
А этот комментарий почему не отработали. Просьба отвечать на каждый комментарий, чтобы было понятно, что вопрос понятен или принят к работе.

Atom098890 replied:
Так  в коде использовал этот трюк. Хорошо, буду комментировать

korbutds replied:
Точно. Правда не совсем верно. От переменных можно избавиться. Почитайте еще раз статью.

korbutds commented on 9-sort-loops/homework.js:
```diff
@@ -0,0 +1,18 @@
+
+function sortArray(arr) {
+    for (let i = 1; i < arr.length; i++) {
+        for (let j = 0; j < arr.length; j++) {
```
Попробуйте j завязать на i.

korbutds commented on 9-sort-loops/homework.js:
У массивов (объектов) есть понятие мутирование. Это, чаще всего, не самое лучшее решение. Лучше в таких функциях, которые мутируют аргументы, аргументы объекты копировать. Поэтому лучше скопируйте массив, проводите манипуляции над копией, и ее возвращайте.

Atom098890 replied:
Я подумал что нужно именно изначальный массив менять и возвращать. Спасибо за идею

Commit:
Правки pull-request
Переписал на switch...case в 7-converter
Поправил цикл и присваивание переменных в файле 9-sort-loops

Commit:
исправил ошибку в вызове функции

Commit:
Убрал лишнии переменные в файле 9-sort-loops
